Ngception/journey-monorepo
# JRNY-168 | Update NX dependencies

# Changes
- Updated and migrated NX dependencies to `14.7.5`
- Updated Jest from version `27` to `28`

# Screenshots
N/A

# Checklist

# Testing steps
- Checkout branch & start db/apps
- Verify everything still works
